### ISSUE TITLE
Able to send packets but exploit doesnt activate.

### ISSUE THREAD

**Issue by Destroxz** [yellows8/3ds_smashbroshax#4]
I have a N3DS and was forced to update to 10.3.0-28U, so does smashhax work on 10.3 or does it still have to be updated? Thank you! ( smash bros is on ver 1.1.2 eshop version)

**Comment by yellows8** [comment]
No reason why not if you use the right payload -> http://3dbrew.org/wiki/10.3.0-28

**Comment by Destroxz** [comment]
I noticed that in terminal it says that the mac address doesnt match the specified mac address. ( I am using the the smashhaxEZ method that Cydget put together and the new payload)

**Comment by yellows8** [comment]
That's normal.

**Comment by Destroxz** [comment]
Oh okay, then I believe I am doing everything right, the packets reach about 12k and still nothing :( Thanks anyways!

**Comment by yellows8** [comment]
Are you broadcasting on channel 6?

**Comment by Destroxz** [comment]
I am new to Linux I don't know what that means :S I only followed the tutorial that cydget put out.

**Comment by yellows8** [comment]
https://github.com/yellows8/3ds_smashbroshax#usage-guide

**Comment by yellows8** [comment]
Works fine here with gamecard Smash v1.1.2 + New3DS 10.3.0-28U.

**Comment by Destroxz** [comment]
I just did the full tutorial now but it still doesnt activate I dont know why. :(

**Comment by Destroxz** [comment]
Also this time I dont have the mac address thing, I installed the new starter kit on my sd card along with 10.3 otherapp payload and renamed to smashpayload.bin and I am successful at sending packets but my game doesnt act when I open group.

**Comment by yellows8** [comment]
How long did you let your system at the group session-scanning menu?

**Comment by Destroxz** [comment]
Sorry I was mistaken the mac still says doesnt match, but that is normal as u said.

**Comment by Destroxz** [comment]
The most I left it is until 12 or 16k packets at 9-10pps. Cant remeber if 12 or 16k. Well over 5 minutes I think maybe even 10.

**Comment by yellows8** [comment]
Maybe try capturing a frame being broadcasted, then upload it.

**Comment by Destroxz** [comment]
Sorry, but what do you mean by capturing a frame? :S

**Comment by yellows8** [comment]
Use Wireshark/whatever.

**Comment by Destroxz** [comment]
I downloaded wireshark but i dont know how to use it. :S

**Comment by Destroxz** [comment]
![1447175665386-594948366](https://cloud.githubusercontent.com/assets/15775903/11069446/95ce504c-87ef-11e5-8664-bdc13789983e.jpg)

Thats what I did I hope it's right.

**Comment by yellows8** [comment]
Save that pcap and upload it.

**Comment by Destroxz** [comment]
![frame](https://cloud.githubusercontent.com/assets/15775903/11069657/227d6b8e-87cf-11e5-8a8c-08ebcde397ee.png) I changed the extension because I cant attach pcapng, so just change the extension to open it.

**Comment by yellows8** [comment]
Looks fine to me, can't think of anything that could be wrong other than broadcasting on the wrong wifi channel(can't tell what channel you're using by the pcap).

```
 sudo iwconfig wlan0 channel 6
```

**Comment by Destroxz** [comment]
I did use channel 6, is putting sudo necessary even if I am root?

**Comment by yellows8** [comment]
No.

"I did use channel 6" :-/

**Comment by Destroxz** [comment]
It's really weird is there something I am doing wrong? Oh well, this was my only way to get homebrew, looks like I will have to wait for a new vuln to be found in 10.3. :/ does it have to do with my internal wifi card? Or as long as packets are being sent then everything should be fine?

**Comment by yellows8** [comment]
"can't think of anything that could be wrong"

**Comment by Destroxz** [comment]
Oh well, thanks a ton for your time, really appreciate it! :)

**Comment by Destroxz** [comment]
Its weird though, because a guy with and o3ds 10.3 managed to get the payload to run but then crashes to home menu, and I am here with a N3ds and not even payload running :S

**Comment by Destroxz** [comment]
Btw, (i know thats not the place for this but)I also tried getting a prepatched oot3dhax save when I was on 10.2 and it says data corrupted and formats.

**Comment by yellows8** [comment]
" prepatched oot3dhax save when I was on 10.2 and it says data corrupted and formats." <- Wait for the release-archive @ oot3dhax repo later I guess.

**Comment by Destroxz** [comment]
Okay :) oh yeah and by prepatched I meant that I got a save file from someone that had oothax and recovered the save to mmy cartridge using powersaves

**Comment by profi200** [comment]
Not every WiFi dongle/card is compatible or works well with smashbroshax. With my build in WiFi card it worked directly for example.

**Comment by Destroxz** [comment]
Yeah, but if it sends it means that its working, because when I use the usb dongle it doesnt work, but internal wifi works.

**Comment by Kudinchou** [comment]
I have a similar problem, when you run aireplay-ng charged 10 packets per second, but when loading with smashbros - group does not load the homebrew launcher (PS: I have a usademo and OLD 3DS 10.3.28U, or is it impossible to work in the OLD 3DS?)

**Comment by ihaveamac** [comment]
smashbroshax doesn't load the *hax/homebrew launcher payload on Old 3DS.

**Comment by Kudinchou** [comment]
😭 thanks friend, and no update to the version browserhax to 1.7626?

**Comment by benoittachet** [comment]
I have the same problem, with a N3DS 11.2.0-35E and SSB 1.0.1
I've seen two or three tutorials on internet, including the one in the repo's readme, all differents, and no one working...
I have extract the homebrew starter kit at my SD's root, put the payload at the same place, and launched aireplay-ng on my wifi device, but nothing comes...
I saw in the readme that we have to use ctr-wlanbeacontool, but I didn't found how...
Thank you for your answers, everyone.
